src/transformers/tokenization_bert_japanese.py

```diff
@@ -164,7 +168,7 @@ def _tokenize(self, text):
 class MecabTokenizer(object):
     """Runs basic tokenization with MeCab morphological parser."""
 
-    def __init__(self, do_lower_case=False, never_split=None, normalize_text=True):
+    def __init__(self, do_lower_case=False, never_split=None, normalize_text=True, mecab_option=""):
```

Comment:
maybe a `Optional[str] = None` would be cleaner here?

Reply:
Fixed!